src/explore-education-statistics-admin/src/services/release/edit-release/data/editReleaseDataService.ts

```diff
@@ -218,8 +218,14 @@ const editReleaseDataService = {
     );
   },
 
-  async deleteChartFile(releaseId: string, fileName: string): Promise<null> {
-    return client.delete<null>(`/release/${releaseId}/chart/${fileName}`);
+  async deleteChartFile(
+    releaseId: string,
+    fileName: string,
+    subjectName: string,
```

Comment:
Would put `subjectName` before `fileName` so that the parameter ordering matches the actual URL.

Reply:
Agreed - done - Thanks